example/cmd/device-simple/Attribution.txt

```diff
@@ -158,3 +158,6 @@ https://github.com/hashicorp/errwrap/blob/master/LICENSE
 
 hashicorp/go-multierror (Mozilla Public License 2.0) https://github.com/hashicorp/go-multierror
 https://github.com/hashicorp/go-multierror/blob/master/LICENSE
+
+github.com/go-kit/log (MIT) https://github.com/go-kit/log
+https://github.com/go-kit/log/blob/master/LICENSE
```

Comment:
could you please also remove the section of 
```
go-kit/kit (MIT) https://github.com/go-kit/kit
https://github.com/go-kit/kit/blob/master/LICENSE
```

Reply:
@AlexCuse,  so it looks like `go-kit/log` replaced use of `go-kit/`